src/fides/api/util/consent_util.py

```diff
@@ -661,6 +661,7 @@ def create_default_experience_config(
     PrivacyNoticeRegion.sk,
     PrivacyNoticeRegion.fi,
     PrivacyNoticeRegion.se,
+    PrivacyNoticeRegion.gb,
```

Comment:
Nothing to do here, this is just a note to self: check if post-translation work we still want "gb" to be shown TCF Experiences.

I realize GB isn't in the EEA and I can't remember if we still added gb_* anyway because we wanted them to still see TCF Experiences.  This list of EEA countries is going away in the new translation work, but this is just a note to make sure our post-translation changes that we still have the right regions we want on TCF.